net/nginx/files/nginx.init

```diff
@@ -75,6 +75,40 @@ nginx_init() {
 	logger -t "nginx_init" -p "daemon.info" "using ${CONF} (the test is ok)"
 }
 
+add_mdns() {
+        local cfg="$1"
+        local port enabled mdns service
+
+        config_get enabled "$cfg" enabled
+        config_get mdns "$cfg" mdns
+        config_get port "$cfg" listen
+
+        port=$(echo "$port" | head -n1 | awk '{print $1;}')
+
+        if [ "${mdns}" == "0" ] || [ "${mdns}" == "false" ]; then
```

Comment:
Also, do not use `==` but `=` (here and elsewhere)

Reply:
Looks like = is the standard in POSIX, updated.